composer.json

```diff
@@ -24,7 +24,7 @@
         "elife/api-client": "^1.0@dev",
         "elife/api-sdk": "dev-master",
         "elife/civi-contacts": "dev-master",
-        "elife/patterns": "dev-master",
+        "elife/patterns": "dev-feature/tabbed-navigation",
```

Comment:
This will need to be changed back to dev-master once patterns-php PR has been merged in.

Reply:
yes, waiting for @chris-huggins approval, then I'll change it to `dev-master`